src/components/footer/socialLinksBlock/LinksList.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import classes from "./LinksList.module.css";
+import { Link } from "react-router-dom";
+
+const LinksList: React.FC<{
```

Comment:
Create interface
Destructuring for props

Reply:
Done as well, thanks